metadata-service/factories/src/main/resources/application.yml

```diff
@@ -108,6 +108,9 @@ visualConfig:
   assets:
     logoUrl: ${REACT_APP_LOGO_URL:/assets/platforms/datahublogo.png}
     faviconUrl: ${REACT_APP_FAVICON_URL:/assets/favicon.ico}
+  entityProfile:
+    # we only support default tab for domains right now. In order to implement for other entities, update React code
+    domainDefaultTab: ${DOMAIN_DEFAULT_TAB:} # set to DOCUMENTATION_TAB to show documentation tab first
```

Comment:
are we intentionally leaving the right side null? : 

Or should we just do ${DOMAIN_DEFAULT_TAB}

Reply:
yeah we want to intentionally leave this null for the default case! then in saas we'll populate this with `DOCUMENTATION_TAB` for Robinhood only.

if we just do `${DOMAIN_DEFAULT_TAB}` without the trailing `:` then the graphql response looks like: `domainDefaultTab: "${DOMAIN_DEFAULT_TAB}"` - not the worst thing and results in the same outcome, but is more unexpected than receiving `""` for the default case imo.